index.html

```diff
@@ -0,0 +1,237 @@
+                    <h4 class="main__price-container-variant-number">700 $</h4>
+                    <ul class="main__price-container-variant-dignities">
+                        <li class="main__price-container-variant-dignities-paragraph">One or two locations</li>
+                        <li class="main__price-container-variant-dignities-paragraph">200 photos in color</li>
+                        <li class="main__price-container-variant-dignities-paragraph">20 photos in retouch</li>
+                        <li class="main__price-container-variant-dignities-paragraph">Readiness 1-2 weeks</li>
+                        <li class="main__price-container-variant-dignities-paragraph">Make up, visage</li>
+                    </ul>
+                    <button class="main__price-container-variant-button">Order shooting</button>
+                </section>
+                <section class="main__price-container-variant">
+                    <h3 class="main__price-container-variant-name">Gold</h3>
+                    <h4 class="main__price-container-variant-number">1000 $</h4>
+                    <ul class="main__price-container-variant-dignities">
+                        <li class="main__price-container-variant-dignities-paragraph">Three locations or more</li>
+                        <li class="main__price-container-variant-dignities-paragraph">300 photos in color</li>
+                        <li class="main__price-container-variant-dignities-paragraph">50 photos in retouch</li>
+                        <li class="main__price-container-variant-dignities-paragraph">Readiness 1 week</li>
+                        <li class="main__price-container-variant-dignities-paragraph">Make up, visage, hairstyle</li>
+                    </ul>
+                    <button class="main__price-container-variant-button">Order shooting</button>
+                </section>
+
+            </div>
+        </section>
+        <section class="main__feedback" id="main-feedback">
+            <form action="" class="main__feedback-form" id="main__feedback_form">
+                <h2 class="main__feedback-form-title">Contact me</h2>
+                <input type="text" class="main__feedback-form-input-email" placeholder="E-mail"
+                    id="main__feedback_form_input_email">
+                <input type="text" class="main__feedback-form-input-phone" placeholder="Phone"
+                    id="main__feedback_form_input_phone">
+                <textarea class="main__feedback-form-input-message" cols="1" rows="3" placeholder="Message"
+                    id="main__feedback_form_input_message"></textarea>
+                <button class="main__feedback-form-send-button" id="main-feedback-form-send-button">Send
+                    message</button>
+            </form>
+        </section>
+    </main>
+    <footer class="footer">
+        <section class="footer__data">
+            <p class="footer__data-element">©</p>
```

Comment:
We need to use paragrphs for some big amount of text (usually on full width of page), not for some short labels. Span is better here.